1. 有关tf.image.decode_image问题
The basic problem is that the output of tf.image.decode_image cannot be passed to tf.image.resize_images.
It raises ValueError: 'images' contains no shape. in the call to resize_images.

It's important to pass expand_animations = False as an argument:

Try:
`tf.image.decode_image(img, expand_animations=False)`